_posts/2020-11-12-quarkiverse.adoc

```diff
@@ -0,0 +1,66 @@
+---
+layout: post
+title: Quarkiverse - The Harbour of Community Extensions
```

Comment:
```suggestion
title: Quarkiverse - The Hub of Community Extensions
```
How about a hub instead of harbour?

Reply:
That makes sense. I think we never discussed or agreed on a name anyway :)